apps/framework-docs/src/pages/index.mdx

```diff
@@ -1,46 +1,53 @@
-
 # Introduction
+
 Welcome to the MooseJS documentation! MooseJS is still in early development, and we're talking to users and potential users all the time to make sure we're creating something valuable. If you're at all interested in what we're building here, then we'd love to talk to you! Please join our [Slack community](https://join.slack.com/t/moose-community/shared_invite/zt-2fjh5n3wz-cnOmM9Xe9DYAgQrNu8xKxg) and reach out. We can't wait to hear from you!
 
 ---
 
-
-
 ## What is MooseJS?
-MooseJS is a JavaScript/Typescript framework for building data-intensive applications and services. Just like you use frontend application frameworks like NextJS to build your frontend applications, you can use MooseJS to build your data/analytics stack. MooseJS is designed to be intuitive and scalable for software engineers who do not have a background in data engineering. You can combine MooseJS and your favorite frontend framework to build feature-rich, interactive data applications for your users and stakeholders.
+
+MooseJS is a JavaScript/Typescript framework for building data-intensive applications and services. Support for more languages is coming soon! Just like you use frontend application frameworks like NextJS to build your frontend applications, you can use MooseJS to build your data/analytics stack. MooseJS is designed to be intuitive and scalable for software engineers who do not have a background in data engineering. You can combine MooseJS and your favorite frontend framework to build feature-rich, interactive data applications for your users and stakeholders.
```

Comment:
I added ` Support for more languages is coming soon!` because although we do mention that later in the docs - I didn't want a non-javascript developer to give up too early on us.  Adding the statement at least asked that they stick with us.